templates/All/Default.html

```diff
@@ -1,44 +1,36 @@
 <div class="coveo-result-frame">
-  <div class="coveo-result-row">
-    <div class="coveo-result-cell" style="width:85px;text-align:center;padding-top:7px;">
-      <span class="CoveoIcon">
-      </span>
-      <div class="CoveoQuickview">
+  <div class="coveo-result-cell" style="vertical-align:top;text-align:center;width:32px;">
```

Comment:
Although I would suggest that this may not be part of the product, and could be something someone else develops as a "theme" for the jsui and that's it. Because it would have to be maintained and the jsui would have no contract to not potentially break it with a new release.